src/Microsoft.IdentityModel.Tokens/IConfigurationRetrievalTime.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    // L2 TODO: internal until L2 cache is implemented S2S.
+    internal interface IConfigurationRetrievalTime
```

Comment:
Prefer verbs when it makes sense, something like: IConfigurationTimeRetriever

Reply:
Do we need this interface?

Reply:
It's needed to help figure out the time-to-live (TTL) for the config object since it might come from an L2 cache. ConfigurationManager works with configuration type T (not the base configuration), so config objects made outside the code can also use the interface without inheriting from BaseConfiguration.

Reply:
Updated to `IConfigurationTimeRetriever`, thanks!